Makefile

```diff
@@ -0,0 +1,3 @@
+dev:
+	@echo "[dev] Running":
+	@nodemon --exec go run cmd/api/main.go --signal SIGTERM 
```

Comment:
maybe you can find a other solution for hot reloading implemented in Golang, because `nodemon` script use `nodejs`.

By the other side, if I clone the project now and then I will run `make dev` command the script will be failed because I have not installed `nodemon` on my computer (and it's not specified in a `package.json` file or in install 
 make command.